.github/issue_template.md

```diff
@@ -0,0 +1,7 @@
+### Hail uses issues to track bugs. For support and feature requests, please use the discussion forum: http://discuss.hail.is/
+
+### Hail version used:
```

Comment:
Can you increase the number of "#" on the headers or make them bullets instead? I don't like looking at multiple lines of bold font sequentially.